boards/espressif/esp_wrover_kit/esp_wrover_kit_esp32_procpu.dts

```diff
@@ -187,6 +189,35 @@
 	status = "okay";
 };
 
+&sdhc {
+	status = "okay";
+	max-bus-freq = <52000000>;
+
+	slots {
+		slot1: sdhc-slot@1 {
+			reg = <0x1>;
+			status = "okay";
+
+			pinctrl-0 = <&sdhc_slot1_default>;
+			pinctrl-names = "default";
+			power-delay-ms = <100>;
+			bus-width = <4>;
+
+			clk-pin = <14>;
+			cmd-pin = <15>;
+			d0-pin = <2>;
+			d1-pin = <4>;
+			d2-pin = <12>;
+			d3-pin = <13>;
+		};
+	};
+
+	mmc {
```

Comment:
Should this mmc node be declared under slot 1? I would expect it to be declared as the child of the node implementing support for the SDHC API